jsettlers.logic/src/main/java/jsettlers/logic/map/loading/original/data/EMapObjectTypeType.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
```

Comment:
Please rename the class to EOriginalMapObjectType to distinguish it from our own EMapObjectType. The same goes for EMapNations, EMapBuildingType, EMapResources, EMapSettlersType, ...